.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
+
+*.b64 binary
```

Comment:
The b64 file is a text-format (ASCII) representation of binary content, so I think this change should be reverted

Reply:
This setting tells git not to add new lines to the file or try to display diffs of it as if it is a normal file (which it can't do effectively, as it's all on one line anyway).

See the **Identifying Binary Files** section of https://git-scm.com/book/ms/v2/Customizing-Git-Git-Attributes , which states:

> Some files look like text files but for all intents and purposes are to be treated as binary data.

and then goes on to describe an example of a text file that is not mergeable and where diffs aren't "generally helpful, advising that in these cases the files in question should be labeled `binary` in `.gitattributes`

It later describes how to get `git` to display diffs of "binary" files that have been converted to text, which we could do, but that requires a `git config` setting, which would have to be set for every user who clones the repo, so it's not really practical.